src/main/java/cart/business/MemberService.java

```diff
@@ -0,0 +1,23 @@
+package cart.business;
+
+import cart.data.MemberRepository;
+import cart.presentation.dto.ViewMember;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class MemberService {
+
+    private final MemberRepository memberRepository;
+
+    public MemberService(MemberRepository memberRepository) {
+        this.memberRepository = memberRepository;
+    }
+
+    public List<ViewMember> getMembers() {
+         return memberRepository.getMembers().stream().map(ViewMember::new)
```

Comment:
메서드 체이닝을 할 때는 각 메서드마다 개행해주는 것이 가독성이 좋다고 생각해요~

```suggestion
         return memberRepository.getMembers()
             .stream()
             .map(ViewMember::new)
```